e2e/tests/cluster_restart.robot

```diff
@@ -15,27 +17,27 @@ ${CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS}    0
 
 *** Test Cases ***
 Restart Cluster While Workload Heavy Writing
-    Given Create deployment 0 with rwo volume
-    And Create deployment 1 with rwx volume
-    And Create deployment 2 with rwo and strict-local volume
-    And Create statefulset 0 with rwo volume
-    And Create statefulset 1 with rwx volume
-    And Create statefulset 2 with rwo and strict-local volume
+    Given Create deployment deploy-0 using rwo volume
```

Comment:
Suggest keeping `0` as before? `deploy-0` seems like just a convention, so you can use the resource type `deployment` as the prefix to make the name.